templates/_layouts/docs.html

```diff
@@ -126,6 +126,9 @@ <h4 class="p-table-of-contents__header">On this page:</h4>
       window.Prism = window.Prism || {};
       Prism.manual = true;
     </script>
+    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.15.1/beautify.min.js" integrity="sha512-kElKX79OYM/vf2AU+lF4Zsi3AnKds70Mc6fRBImpwQzdZNbju6SyeTBBOUe1N9q1P456v1F01HAsjjWZkXIS4Q==" crossorigin="anonymous"></script>
```

Comment:
Would be good to copy those to our static folder, not to rely on external resources.

Reply:
Updated to copy to static folder

Reply:
No, it's fine as long as they are somewhere in our static files. Good to keep it consistent with the rest of files we are copying over from node modules.